tests/data/snake_oil_data.py

```diff
@@ -140,41 +140,41 @@ def content(self):
         "name": 0,
         "parameters": [
             {
-                "data_url": "http://127.0.0.1:5000/data/33",
+                "data": "http://127.0.0.1:5000/data/33",
                 "name": "BPR_138_PERSISTENCE",
             },
             {
-                "data_url": "http://127.0.0.1:5000/data/58",
+                "data": "http://127.0.0.1:5000/data/58",
                 "name": "BPR_555_PERSISTENCE",
             },
             {
-                "data_url": "http://127.0.0.1:5000/data/83",
+                "data": "http://127.0.0.1:5000/data/83",
                 "name": "OP1_DIVERGENCE_SCALE",
             },
         ],
         "responses": [
             {
-                "data_url": "http://127.0.0.1:5000/data/284",
+                "data": "http://127.0.0.1:5000/data/284",
                 "name": "SNAKE_OIL_GPR_DIFF",
             },
             {
-                "data_url": "http://127.0.0.1:5000/data/310",
+                "data": "http://127.0.0.1:5000/data/310",
                 "name": "SNAKE_OIL_OPR_DIFF",
             },
             {
-                "data_url": "http://127.0.0.1:5000/data/336",
+                "data": "http://127.0.0.1:5000/data/336",
                 "name": "SNAKE_OIL_WPR_DIFF",
             },
         ],
     },
     "http://127.0.0.1:5000/ensembles/1/responses/SNAKE_OIL_GPR_DIFF": {
         "alldata_url": "http://127.0.0.1:5000/ensembles/1/responses/SNAKE_OIL_GPR_DIFF/data",
-        "axis": {"data_url": "http://127.0.0.1:5000/data/283"},
+        "axis": {"data": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]},
         "ensemble_id": "1",
         "name": "SNAKE_OIL_GPR_DIFF",
         "realizations": [
             {
-                "data_url": "http://127.0.0.1:5000/data/284",
+                "data": "http://127.0.0.1:5000/data/284",
                 "name": 0,
                 "ref_url": "http://127.0.0.1:5000/ensembles/1/realizations/0",
                 "summarized_misfits": {},
```

Comment:
data is here a URL

Reply:
Yes i forgot to change the values on the data not being used ... fixed for those used in the test... Now i replaced all the urls with just dummy data - i'm not sure how to approach this with a sane test data set - but that we can discuss later 👍